cmd/operator-sdk/bundle/create_test.go

```diff
@@ -0,0 +1,145 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bundle
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Checking operator-sdk bundle create command", func() {
+	Describe("newCreateCmd", func() {
+		It("builds and returns a cobra command", func() {
+			cmd := newCreateCmd()
+			Expect(cmd).NotTo(BeNil())
+
+			flag := cmd.Flags().Lookup("directory")
+			Expect(flag).NotTo(BeNil())
+			Expect(flag.Shorthand).To(Equal("d"))
+
+			flag = cmd.Flags().Lookup("output-dir")
+			Expect(flag).NotTo(BeNil())
+
+			flag = cmd.Flags().Lookup("tag")
+			Expect(flag).NotTo(BeNil())
+			Expect(flag.Shorthand).To(Equal("t"))
+
+			flag = cmd.Flags().Lookup("package")
+			Expect(flag).NotTo(BeNil())
+
+			flag = cmd.Flags().Lookup("channels")
+			Expect(flag).NotTo(BeNil())
+			Expect(flag.Shorthand).To(Equal("c"))
+			Expect(flag.DefValue).To(Equal("stable"))
+
+			flag = cmd.Flags().Lookup("generate-only")
+			Expect(flag).NotTo(BeNil())
+			Expect(flag.DefValue).To(Equal("false"))
+
+			flag = cmd.Flags().Lookup("overwrite")
+			Expect(flag).NotTo(BeNil())
+			Expect(flag.DefValue).To(Equal("false"))
+
+			flag = cmd.Flags().Lookup("image-builder")
+			Expect(flag).NotTo(BeNil())
+			Expect(flag.Shorthand).To(Equal("b"))
+			Expect(flag.DefValue).To(Equal("docker"))
+
+			flag = cmd.Flags().Lookup("default-channel")
+			Expect(flag).NotTo(BeNil())
+			Expect(flag.Shorthand).To(Equal("e"))
+		})
+	})
+
+	Describe("validate", func() {
```

Comment:
Hi @jberkhahn, @estroz   

I do not think that the validate should be in the `create_test.go` since if we follow up the standard(go common practices) we will have the tests from create.go in the create_test.go and the test of validate.go in the validate_test.go.

Any reason for we do not follow up the common practice in this case?